docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.paragraph.field_card_layout_display.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_card_layout_display
```

Comment:
This name needs to change. @dlescarbeau this one is not going to be on you. 

Once again this is a generically named field with a very specific set of options. So either the allowed values should change to not be cgdp-imageless-card-group OR this field needs to change. 
@andyvanavery31

Reply:
@dlescarbeau, Bryan and I chatted and we are going to change the field name to field_imgless_crd_grp_display. This should be within the 30 character limit

Reply:
@bryanpizzillo confirming this one still isn't going to be on me after your chat with @andyvanavery31 ? Otherwise you and I may want to connect